### ISSUE TITLE
AT1 Remark 3.1.7

### ISSUE THREAD

**Issue by jkostiuk** [TeamBasedInquiryLearning/linear-algebra#271]
refers to additive identity prior to it being defined in AT5.

**Comment by siwelwerd** [comment]
I guess here and in [Definition 3.1.1](https://teambasedinquirylearning.github.io/linear-algebra/2023/AT1.html#AT1-definition-map) , we are still referring to 'vector spaces' $V$ and $W$, but have not defined what a vector space is at this point.  So maybe both should be changed to $\mathbb{R}^m$ and $\mathbb{R}^n$ to avoid this, and then it is fine to just use $\vec{0}$ in 3.1.7 without the parenthetical.

Assigning this to @StevenClontz since it is related to the reorganization he did that I've only glanced over.

**Comment by StevenClontz** [comment]
So, I purposely pulled a fast one here. Look at this:

[Observation 3.6.1](https://teambasedinquirylearning.github.io/linear-algebra/2023/AT6.html#observation-17) 

I don't think a single student has stood up and asked me what a vector space is because I haven't technically defined it. As that happened to you Jordan? They just assume I mean a Euclidean vector space, and there's nothing illegal about letting $V=\mathbb R^n$ and $W=\mathbb R^m$ implicitly.

On the other hand, I want to be more careful about referring to $\vec 0$ and $\vec z$ so things stay technically correct. I was indeed asked this semester if $\vec 0$ meant $[0 0 0]^T$.

**Comment by jkostiuk** [comment]
Not in my section I don’t think, but I have tripped up myself referring to “vector spaces” and it think it might’ve thrown off a few other facilitators lol. Cheers,JordanOn Oct 19, 2023, at 2:24 PM, Steven Clontz ***@***.***> wrote:﻿
So, I purposely pulled a fast one here. Look at this:
Observation 3.6.1
I don't think a single student has stood up and asked me what a vector space is because I haven't technically defined it. As that happened to you Jordan? They just assume I mean a Euclidean vector space, and there's nothing illegal about letting $V=\mathbb R^n$ and $W=\mathbb R^m$ implicitly.
On the other hand, I want to be more careful about referring to $\vec 0$ and $\vec z$ so things stay technically correct. I was indeed asked this semester if $\vec 0$ meant $[0 0 0]^T$.

**Comment by siwelwerd** [comment]
I was noticing some other places in Chapter 2 where we refer to "spaces" rather than "vector spaces".  So I am wondering this morning if a good solution is to expand [Note 2.1.1](https://teambasedinquirylearning.github.io/linear-algebra/2023/EV1.html#note-1) to say that Euclidean spaces are examples of a more general thing called a 'vector space', which is explored in Section 3.5.  And go ahead and explain the "fast one"--we are going to do a whole bunch of things that also apply to vector spaces, but focus on Euclidean spaces. So we will use the word "vector space" when the statement is more general, but readers can, if they wish, assume in this chapter that this always means $\mathbb{R}^n$ for some $n$.

**Comment by jkostiuk** [comment]
i was talking to drew about this earlier this morning and wanted to share my comments here--I'm in the camp of liking the fact that we're using the word vector space and then hoodwinking them. I had a really fun time today doing a "is a hotdog a sandwich" activity to motivate AT4. My students understood that a big part of this debate is that we haven't agreed on what defines a sandwich or what defines a hotdog. I closed class by pointing out that we also have not defined what a vector is and that really made them think. 

My vote would be to define a Euclidean Spaces and then keep using the word vector spaces and the symbols $V$ and $W$. I think the pay off will be really good tomorrow.

**Comment by siwelwerd** [comment]
So the original issue about the use of the phrase additive identity was actually fixed in 2bfdf82d .  

I added a couple `<aside>`s in #292 .  If y'all like how those look, I would propose then going through and changing any use of "space" to say either "Euclidean space" or "vector space" as appropriate.  The intention is to kind of keep the reveal for the students, while adding in some signposting for an instructor/individual reader.

Somewhat tangential ~~aside~~  ~~note~~ comment: Note 2.1.1 currently does not build into the slides, as it is a `<note>`.  If this is not the desired behavior for `<note>`s, lets make a separate issue.

**Comment by siwelwerd** [comment]
Still waiting on feedback here before moving forward. @StevenClontz @jkostiuk

**Comment by StevenClontz** [comment]
> Note 2.1.1 currently does not build into the slides

Is this true with the refactored slide build?

**Comment by siwelwerd** [comment]
Yes. It should be an easy fix if that is not the desired behavior, but create a separate issue for that.

**Comment by StevenClontz** [comment]
I think I like asides not building into slides, but every numbered "block" should get a slide I think.

**Comment by siwelwerd** [comment]
This was closed by #292
